README.md

```diff
@@ -22,6 +22,12 @@ How to run the development server:
 % bundle exec hanami server
 ```
 
+How to run the code reloader:
+
+```
+% bundle exec hanami reloader
```

Comment:
Perhaps we should make it clear that the server needs to be running in order for this command to work

Reply:
It doesn't, sounds like there's only something wrong in your environment. Are you getting any errors?